routes/mylibrary.js

```diff
@@ -18,9 +18,8 @@ const selectAllBooks = `SELECT UserBooks.id AS userBookId, Books.id AS bookId, a
                         INNER JOIN Books ON Books.id = UserBooks.bookId
                         WHERE UserBooks.userId = ?`;
 const deleteUserBook = 'DELETE FROM UserBooks WHERE id = ?';
```

Comment:
I think these should remain variables as they are not really string constants since values are placed in them to change them. Unless the query is used multiple times throughout the route, they should also be moved into the function they are used in for easier readability, having to scroll to the top to see what query and what the values being sent in are actually replacing is an unnecessary pain point.